sdk/iot/Azure.Iot.Hub.Service/src/Generated/Models/FaultInjectionConnectionProperties.cs

```diff
@@ -16,15 +16,17 @@ public FaultInjectionConnectionProperties()
         }
```

Comment:
Should we make all FaultInjection models internal?

Reply:
Yes this should be internal. I believe we need to still evaluate what all needs to be internal.  Should I request for removal of fault injection from swagger or do we need it for testing?

Reply:
we don't need it for testing, since we have a FI setup in place already, one that works on top of our existing hub device messaging APIs. This is safe to remove from our swagger.

Reply:
We've already heard from Andrei that they didn't want us to expose it unless we had a compelling reason. Let's just make it and all things related to it internal.